app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,38 @@
  *= require_tree .
  *= require_self
  */
+
+ body {
+ font-family: 'Tenor Sans', sans-serif;
+ background-color:  	#f7f7f7;
+ }
+
+ .p1{border-width:10px;border-style:solid;padding:5px;border-color:green;}
```

Comment:
Can we insert some line breaks please?

Reply:
Artefact left over from testing out borders. Removed.